app/policies/issue_policy.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+class IssuePolicy < ApplicationPolicy
+  def new?
+    !user.nil? && user.reporter?
+  end
+
+  def create?
+    !user.nil? && user.reporter?
```

Comment:
@igorkasyanchuk as per our discussion with guys we decided, that only people that logged in via Facebook should be able to post an issue. Admin logs in via hidden auth page with his login and password